keyboards/kprepublic/bm40hsrgb/keymaps/dan/config.h

```diff
@@ -0,0 +1,7 @@
+// Copyright 2021 Your Name (@syntax-magic)
```

Comment:
"Your Name"?  May want to change that. :)

Reply:
Hi drashna, thanks for pointing it out. My name has been added. Cheers.